src/io/flutter/android/AndroidSdk.java

```diff
@@ -138,14 +159,68 @@ private static AndroidSdk fromSdk(@Nullable Sdk candidate) {
     return new AndroidSdk(candidate, home);
   }
 
+  @Nullable
+  public VirtualFile getEmulatorToolExecutable() {
+    // Look for $ANDROID_HOME/tools/emulator.
+    return home.findFileByRelativePath("tools/" + (SystemInfo.isWindows ? "emulator.exe" : "emulator"));
```

Comment:
In newer android SDK (After v25.3.0), the emulator tool was moved to ${ANDROID_HOME}/emulator/emulator(.exe). This should probably be accounted for, since the old emulator tool doesn't start emulators. Throws a `PANIC: Missing emulator engine program for 'x86' CPU.`.